datamade/ccfp-asset-dashboard
# added fixture data + loaddata command to the release

## Overview
- Adds fixtures based on data input by Garret. 
- Configures the fixtures to load with the app's deployment.

This is for #13.

### Notes
When Garret input this data, the project categories were already populated from V1 of the `scripts/release.sh` command (a command which is changed in this PR). That data is based on [the raw CIP file](https://github.com/datamade/ccfp-asset-dashboard/blob/master/raw/2021CIPTablesDRAFT10.28.2020updated_GW_clean.csv). He didn't edit any of the existing categories, so I suspect that this data was correct. We need to follow up with him and make sure these values are what they need. I'll note this in #13 so we don't miss it.

## Testing Instructions
- Visit the `/projects` page: https://ccfp-asset-d-fixture-se-m24gb1.herokuapp.com/projects/
- Add a project. Within the form, you should be able to select the sections that Garret added: "GIS", "Surveyor", "Planning", "Landscape Architecture", and "Building Architecture". 
- In the project detail page, you should be able to choose the zones and districts that Garret added. [The fixtures have all the data](https://github.com/datamade/ccfp-asset-dashboard/blob/fb0cbc820e0d213e74c567d213474d089cf6d780/asset_dashboard/fixtures/data.json) if you want to double check.
